server/app/services/grid_service_instance_deployer.rb

```diff
@@ -133,7 +133,7 @@ def get_service_instance
 
   # @param [HostNode] node
   # @raise [RpcClient::TimeoutError]
-  def notify_node(node, timeout: 2.0)
+  def notify_node(node, timeout: 5.0)
     rpc_client = RpcClient.new(node.node_id, timeout)
     rpc_client.request('/service_pods/notify_update', [])
```

Comment:
What if this was `rpc_client.notify` instead of `request`? No need for a timeout in that case, and the agent RPC handler doesn't return or raise anything useful.